lib/methods/import.js

```diff
@@ -682,7 +705,7 @@ module.exports = self => {
         } else {
           await manager.update(_req, docToUpdate);
         }
-        if (doc.aposMode === 'draft') {
+        if (!importDraftsOnly) {
```

Comment:
not sure

Reply:
Probable for both, would keep the first option (working for normal import on draft docs and on draft only mode)